src/buildblock/ML_norm.cxx

```diff
@@ -1199,19 +1224,29 @@ void set_fan_data_add_gaps(ProjData& proj_data,
                     int rb = 0, b = 0;
 
                     proj_data_info_ptr->get_det_pair_for_bin(a, ra, b, rb, bin);
-                    
-                    (*segment_ptr)[bin.axial_pos_num()][bin.view_num()][bin.tangential_pos_num()] = 1e20;
 
-                    
-                    if ((ra+1)% num_axial_crystals_per_block == 0) continue;
-                    if ((rb+1)% num_axial_crystals_per_block == 0) continue;
-                    if ((a+1)% num_transaxial_crystals_per_block == 0) continue;
-                    if ((b+1)% num_transaxial_crystals_per_block == 0) continue;
-                    
-                    int new_a = a - a/num_transaxial_crystals_per_block;
-                    int new_b = b - b/num_transaxial_crystals_per_block;
-                    int new_ra = ra - ra/ num_axial_crystals_per_block;
-                    int new_rb = rb - rb/num_axial_crystals_per_block;
+                    (*segment_ptr)[bin.axial_pos_num()][bin.view_num()][bin.tangential_pos_num()] = 0;
```

Comment:
I suggest we use an extra input argument to the function `gap_value`, defaulting to 0. It will also make it clearer what's happening here.